WasatchNET/Spectrometer.cs

```diff
@@ -142,8 +166,35 @@ public string model
         // internal driver attributes (no direct corresponding HW component)
         ////////////////////////////////////////////////////////////////////////
 
+        // make const?  Do subclasses change this?
         bool throwawayADCRead { get; set; } = true;
 
+        /// <summary>
+        /// Whether the driver should automatically generate a throwaway 
+        /// "stability" measurement after changing integration time.
+        /// </summary>
+        public bool throwawayAfterIntegrationTime { get; set; }

```

Comment:
Previously we were doing the throwaways in WPSC. I suppose now I set it in driver?

Reply:
It's up to you.  Anything "common" that "most people will need to do" I'm trying to automate in the driver, mainly to prevent easily-avoidable errors in customer applications.  We will likely start defaulting that property to "true" for an increasing number of spectrometers, but this is a starting point.